tolumide-ng/EPIC-Mail-Application
# #165565879 Delete Multiple Messages Feature

#### What does this PR do?
Enables the User to check for a specific message and delete multiple messages on epicmail

#### Description of Task to be completed?
Have the endpoint to post messages working
- GET /api/v2/messages/specific/:message-id
- DELETE /api/v2/messages/multiple

#### How should this be manually tested?
- Clone the repo, cd into it, create the database for the test and development environment
- Use the variables in the .env.sample file to create the variables in your local .env file
- Provide details specified in the documentation to create a user (in this case create two different users: a sender and receiver)
- Send multiple messages from user A to user B
- Confirm message has been sent using the get specific message endpoint
- Delete multiple sent messages by using the delete multiple messages endpoint - 
- Supply the following parameters as req.body (**ids: [10, 23], type: 'sent'**), where ids is an array containing the id of the messages to be deleted

#### Any background context you want to provide?
ids must be an array and type must be either 'sent', 'inbox', or 'draft', depending on the type of messages you intend to delete


#### What are the relevant pivotal tracker stories?
#165565879
